sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -725,6 +773,16 @@ private Map<String, String> getDistroVersionsOptionsMap(Set<String> versions, Ve
 		return optionsMap;
 	}
 
+	private Map<String, String> getO3VersionsOptionsMap(VersionsHelper versionsHelper,
```

Comment:
Can you add some javadoc to this method? It would also be great to include some examples of the contents of the returned map.